lib/CircuitBreaker.js

```diff
@@ -43,13 +43,13 @@ function updateCircuitBreakerConfigForImplicitOutputQueue(cbConf, groupId, topic
 const circuitBreakerGauge = ZenkoMetrics.createGauge({
     name: 's3_circuit_breaker',
     help: 'Circuit Breaker State',
-    labelNames: ['type'],
+    labelNames: ['type', 'promErr'],
```

Comment:
the label should not be specific to prometheus : any error could happen...

Reply:
The label has been removed and a new counter metric has been introduced here : https://github.com/scality/backbeat/pull/2516/commits/73b8545ac169e6e00dc3e1ce637db01656c487f2#diff-07825dc8c7adf67b6bbea289468e3446d7646ec1c8f5b9c361b381807fb2ce28R46-R52